RadixWallet/Clients/UserDefaults+Dependency+Extension/UserDefaults+Dependency+Extension.swift

```diff
@@ -138,4 +141,72 @@ extension UserDefaults.Dependency {
 	public func setNPSSurveyUserId(_ id: UUID) {
 		set(id.uuidString, forKey: Key.npsSurveyUserID.rawValue)
 	}
+
+	public var getDidMigrateKeychainProfiles: Bool {
+		bool(key: .didMigrateKeychainProfiles)
+	}
+
+	public func setDidMigrateKeychainProfiles(_ value: Bool) {
+		set(value, forKey: Key.didMigrateKeychainProfiles.rawValue)
+	}
+
+	public var getLastCloudBackups: [UUID: BackupResult] {
+		(try? loadCodable(key: .lastCloudBackups)) ?? [:]
+	}
+
+	public func setLastCloudBackup(_ result: BackupResult.Result, of profile: Profile) throws {
+		var backups: [UUID: BackupResult] = getLastCloudBackups
+		backups[profile.id] = .init(
+			backupDate: .now,
+			profileHash: profile.hashValue,
+			result: result
+		)
+
+		try save(codable: backups, forKey: .lastCloudBackups)
+		print("  •• did setLastCloudBackup \(result)")
```

Comment:
logger?

Reply:
not necessary I think, this is just a leftover